content/client-lib-development-guide/features.textile

```diff
@@ -301,12 +301,12 @@ h3(#realtime-connection). Connection
 * @(RTN3)@ If connection option @autoConnect@ is true, a connection is initiated immediately; otherwise a connection is only initiated following an explicit call to @connect()@
 * @(RTN4)@ EventEmitter and states:
 ** @(RTN4a)@ Implements @EventEmitter@ and emits events for state changes @INITIALIZED@, @CONNECTING@, @CONNECTED@, @DISCONNECTED@, @SUSPENDED@, @CLOSING@, @CLOSED@, @FAILED@
-** @(RTN4g)@ Additionally, an @ERROR@ event can be emitted by the @Connection@ that contains details of the error in an @ErrorInfo@ object
+** @(RTL4h)@ Additionally, an @UPDATE@ event can be emitted by the @Connection@. This is used for changes to connection conditions for which the main connection state (e.g. @CONNECTED@) does not change. (The library must never emit an @RTN4a@ state change event for a state equal to the previous state).
```

Comment:
I am not keen on the terminology "main connection state" i.e. what is main and what is not?  I am struggling to find a better description, but perhaps "This is used for changes to connection conditions for which the `ConnectionState` value does not change..."